tRecorderApi/api/views/__init__.py

```diff
@@ -0,0 +1,20 @@
+from index import *
+from language import *
+from book import *
+from user import *
+from take import *
+from comment import *
+from file_upload import *
+from file_stream import *
+from project import *
+from source_file import *
+from project_zip_file import *
+from project_zip_file_nostore import *
+from exclude_files import *
+from source_file_upload import *
+from all_projects import *
+from project_chapter_info import *
+from push_comments_takes import *
+from version_list import *
+from update_project import *
+from version_list import *
```

Comment:
@mXaln is there a reason that `from version_list import *` is listed 2 times in the imports?